### DISCUSSION TITLE
Update Request

### DISCUSSION THREAD

**Discussion by mak-cs** [jlesage/docker-crashplan-enterprise#9] [General]
Is it possible to update the docker container to the current version?

Greetings
MAK-CS

**Comment by jlesage** [comment]
Yes, I'm working on this!

**Comment by mak-cs** [comment]
Thanks a lot!
Am 25. Mai 2024, 18:28 +0200 schrieb Jocelyn Le Sage ***@***.***>:

Yes, I'm working on this!

**Comment by mak-cs** [comment]
Any news about your update plans? I would be very happy to receive a timeline...

**Comment by jlesage** [comment]
New Docker image now available.

**Reply by mak-cs** [reply]
Thanks a Lot!
